components/Meta.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 the Deno authors. All rights reserved. MIT license.
+import { SITE_DESCRIPTION, SITE_NAME } from "@/constants.ts";
+
+interface MetaProps {
+  title?: string;
+  description?: string;
+  imageUrl?: string;
+  url?: string;
+}
+
+export default function Meta(props: MetaProps) {
+  const { title, description, imageUrl, url } = props;
+
+  return (
+    <>
+      {/* HTML Meta Tags */}
+      <title>{title || SITE_NAME}</title>
+      <meta
+        name="description"
+        content={description || SITE_DESCRIPTION}
+      />
+      <meta property="og:type" content="website" />
+      <meta property="og:site_name" content={title || SITE_NAME} />
+      <meta property="og:locale" content="en" />
```

Comment:
I believe these 3 lines should be moved down to the Facebook Meta Tags section, and `og:site_name` should just have a value of `SITE_NAME`.